app/src/org/commcare/models/database/user/models/AndroidCaseIndexTablePreV21.java

```diff
@@ -4,12 +4,12 @@
 
 import net.sqlcipher.database.SQLiteDatabase;
 
-import org.commcare.CommCareApplication;
 import org.commcare.cases.model.Case;
 import org.commcare.cases.model.CaseIndex;
 import org.commcare.modern.database.DatabaseHelper;
 
-public class AndroidCaseIndexTableV1 {
+// Case Index table extension for Pre user db model 21 to use in DB migration
+public class AndroidCaseIndexTablePreV21 extends AndroidCaseIndexTable {
```

Comment:
What needed to change with AndroidCaseIndex? I don't think it needed to be updated by these changes? (this model is confusingly only about <index> blocks, not about internally indexed properties of the case).

Reply:
That makes sense for the V18 -> V19 migraiton, but why was the change made specifically to extend AndroidCaseIndexTable? To retain access to something needed by the v18->19 process? I think before we were trying to keep the V1 model totally separate so we couldn't accidentally introduce new code to the "Current" model which unexpectedly changed the migration placeholder, so it might be better to add the new needed things to the PreV21 model directly rather than adding the extension.